.github/workflows/build-publish-ondemand.yml

```
@@ -56,6 +56,9 @@ jobs:
           ref: '${{ env.RELEASE }}'
           path: 'ondemand'
 
+      - name: Use the dockerfile from the master branch
+        run: wget -O ondemand/Dockerfile 'https://raw.githubusercontent.com/OSC/ondemand/refs/heads/master/Dockerfile'
```

this seems rather brittle. why not use a commit instead?
```suggestion
        run: wget -O ondemand/Dockerfile 'https://raw.githubusercontent.com/OSC/ondemand/2456c97e81805f1134923a712e8cc4603b77e674/Dockerfile'
```